src/main/java/views/LeagueSimulatorPane.java

```diff
@@ -0,0 +1,220 @@
+package views;
+
+import controllers.SceneManager;
+import javafx.application.Platform;
+import javafx.collections.FXCollections;
+import javafx.collections.ObservableList;
+import javafx.geometry.Pos;
+import javafx.scene.control.Button;
+import javafx.scene.control.ComboBox;
+import javafx.scene.control.Label;
+import javafx.scene.layout.HBox;
+import javafx.scene.layout.VBox;
+import javafx.scene.paint.Color;
+
+import java.util.*;
+
+import javafx.scene.text.Text;
+import models.*;
+
+
+public class LeagueSimulatorPane extends GamePane {
+
+    Game GAME = Game.getInstance();
+
+    private final VBox leftContainer = new VBox();
+    private final VBox matchesContainer = new VBox();
+    private final VBox centerContainer = new VBox();
+
+    private Label title = new Label("League Simulator");
+    private final Button returnButton = new Button("Return");
+    private final Button playButton = new Button("Start");
+
+    ArrayList<String > teamNames = GAME.getAllTeamNames();
+
+    private Label match1 = new Label("Match 1:");
```

Comment:
Will there always be 8 matches regardless of the amount of teams? Consider making this adaptable to an unknown amount of matches.